### PR TITLE
Add feature to communicate over HTTP/Socks proxies

### PR DESCRIPTION
**Pull request by Yuji-Yamamoto-Recruit**

### CODE REVIEW

File: lib/fluent/plugin/output_node.rb [hunk 1]
```diff
@@ -217,8 +221,14 @@ def connect
 
     addr = @sender.hostname_resolver.getaddress(@host)
     log.debug "create tcp socket to node", host: @host, address: addr, port: @port
+
     begin
+    if @proxy_uri.nil? then
```
Comment (tagomoris) [inline lib/fluent/plugin/output_node.rb]: wrong indentation.

**Comment by tagomoris** [comment]
I'm very sorry for late reply.
1. commented about wrong indentation
2. add description to README
2. Japanese commit comment cannot be acceptable. Could you squash commits into a commit with English commit comment?

**Comment by Yuji-Yamamoto-Recruit** [comment]
Thanks for response.

i'm fixed.

# sorry, I'm no good at English...

**Comment by tagomoris** [comment]
Thanks! I squashed your commits and merged.

**Pull request closed without merging**